.gitignore

```diff
@@ -2,6 +2,9 @@ node_modules/
 lib/
 __tests__/runner/*
 
+validate/temp
+validate/node
```

Comment:
I created a bash script to automate interactively validating without queuing a full run